### DISCUSSION TITLE
Return, node, component ve benzeri kelimeleri ingilzice birakalim

### DISCUSSION THREAD

**Discussion by K2adir** [reactjs/tr.react.dev#408] [Ideas]
her nasil, state, hook, string, prop gibi bazi kelimeleri ceviremeden dogrudan kullaniyorsak, bence buna yeni kelimeler eklemeliyiz. 

Returning multiple elements --> Birden Fazla Eleman Döndürme seklinde cevirince ben acikcasi okudugumdan hic birsey anlamiyorum. 

bence buradaki kelimeler listesine eklentiler yapilmali
https://github.com/reactjs/tr.react.dev/blob/main/TRANSLATION.md

**Comment by yunusemredilber** [comment]
@alioguzhan abi selamlar. Hazır yeni çeviriye başlamışken guide’da bir refaktör’e gidebiliriz. İstersen gelen yorumları da dikkate alarak yakın zamanda bir draft çıkartabilirim. Sonrasında onun üzerinden konuşup finalize ederiz.

**Reply by yunusemredilber** [reply]
Bu arada “returning multiple elements” bence çevirilmesi gereken bir yapı. Ama guide’ı yeniden elden geçirmek güzel olabilir.

**Reply by alioguzhan** [reply]
> Bu arada “returning multiple elements” bence çevirilmesi gereken bir yapı.

Buna katiliyorum @yunusemredilber @K2adir 


Baska bir arkadas da ayni konu icin obur basliga yorum yazmisti. O cevabi birakiyorum buraya da:

https://github.com/reactjs/tr.react.dev/discussions/399#discussioncomment-5743828


Ama tabii konusup bi karara varalim hepimiz. Belki farkli fikirler cikar daha guzel bi yontem buluruz. 

Hatta bu baslikta konusulabilir. Orasi kalabaliklasabilir iyice.

**Reply by yunusemredilber** [reply]
@alioguzhan teşekkürler yorumun için, thread’deki yorumlarına katılıyorum. Bence burada çıkan ideoloji’yi de dökümana eklemek gerekli. Popüler terimlerde parantez içinde ingilizce halini kullanmak yada ne seviyede çeviri yapılması gerekitği hakkında daha net bir rule setimiz olabilir.

**Reply by K2adir** [reply]
> Bu arada “returning multiple elements” bence çevirilmesi gereken bir yapı. Ama guide’ı yeniden elden geçirmek güzel olabilir.

birden fazla elemani veya birden fazla element'i return'lemek veya "return etmek" gibi bir cumlenin daha uygun olacagini dusunuyorum. Kimsenin sifirdan React ogrenmeye calisacagini dusunmuyorum. Js'den geliyorsa, zaten return algisindan haberdar olacaktir. Birden fazla element'i return etmek gayet aciklayici olur

**Comment by agmmnn** [comment]
framework'ün çatı olarak kullanımı hiç yaygın değil aynı şekilde framework olarak kalması daha iyi olur. ayrıca interactivity-interaktivite, interaction-interaksiyon olarak da çevirilebilir.

**Reply by anilcanboga** [reply]
Merhabalar, aynı fikirdeyim. Yakın zamanda dokümantasyonun daha yüksek bir kalitede olması için bu güncellemeyi yayına alacağım.
